modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -1042,4 +1042,40 @@ a|The query used a procedure that generated a warning.
 m|WARNING
 |Category
 m|GENERIC
-|===
\ No newline at end of file
+|===
+
+[#_neo_clientnotification_statement_repeatedrelationshipreference]
+=== Neo.ClientNotification.Statement.RepeatedRelationshipReference (when run on version 5.5 or newer)
+
+.Notification category details
+[cols="<1s,<4"]
+|===
+|Code
+m|Neo.ClientNotification.Statement.RepeatedRelationshipReference
```

Comment:
```suggestion
m|Neo.ClientNotification.Statement.RepeatedRelationshipVariableInPattern

Reply:
As suggested by Valerio.

Reply:
Sorry, I just saw that this is an existing code. So I guess we can't change the code and the description.